src/Sepes.RestApi.IntegrationTests/Setup/CustomWebApplicationFactory.cs

```diff
@@ -107,7 +107,8 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
                                [ConfigConstants.IS_INTEGRATION_TEST] = "true",
                                ["AllowCorsDomains"] = "http://localhost:80",
                                ["CostAllocationTypeTagName"] = "INTTEST-CostAllocationType",
-                               ["CostAllocationCodeTagName"] = "INTTEST-CostAllocationCode"
+                               ["CostAllocationCodeTagName"] = "INTTEST-CostAllocationCode",
+                               ["DatasetStorageAccountRoleAssignmentId"] = "9145904e-a892-473a-b559-8d87b9858407" 
```

Comment:
This is the role id, and I guess this is somewhere to be found in azure? Is it in the manifest there?

Reply:
This particular one does not exist anywhere, it just a guid i generated i VS. But in live scenarios, the vaue for DatasetStorageAccountRoleAssignmentId will be set in config, and is a custom role found in Azure